docs/guides/extensions/panels-display-data-vonage.md

```diff
@@ -0,0 +1,769 @@
+```js
+{
+	field: 'direction',
+	name: 'Direction',
+	type: 'string',
+	meta: {
+		width: 'half',
+		interface: 'select-dropdown',
+		options: {
+			choices: [
+				{ text: 'Outbound', value: 'outbound' },
+				{ text: 'Inbound', value: 'inbound' }
+			],
+		}
+	}
+},
+```
+
+It would be useful to control the scope of data for those who transact larger amounts of messages. Add the following
+option for the user to select a range:
+
+```js
+{
+	field: 'range',
+	type: 'dropdown',
+	name: '$t:date_range',
+	schema: { default_value: '1 day' },
+	meta: {
+		interface: 'select-dropdown',
+		width: 'half',
+		options: {
+			choices: [
+				{ text: 'Past 5 Minutes', value: '5 minutes' },
+				{ text: 'Past 15 Minutes', value: '15 minutes' },
+				{ text: 'Past 30 Minutes', value: '30 minutes' },
+				{ text: 'Past 1 Hour', value: '1 hour' },
+				{ text: 'Past 4 Hours', value: '4 hours' },
+				{ text: 'Past 1 Day', value: '1 day' },
+				{ text: 'Past 2 Days', value: '2 days' }
+			]
+		}
+	}
+},
+```
+
+Vonage has the ability to include the message in the response. This will be useful to provide as a preview upon click
+but for larger datasets may impact the performance of the API. Create an option to toggle this on/off:
+
+```js
+{
+	field: 'include_message',
+	name: 'Include Message',
+	type: 'boolean',
+	meta: {
+		interface: 'boolean',
+		width: 'half',
+	},
+	schema: {
+		default_value: false,
+	}
+},
+```
+
+Lastly, add the option to limit the messages to a specific state such as delivered or failed. The default option is
+`Any`:
+
+```js
+{
+	field: 'status',
+	name: 'Status',
+	type: 'string',
+	schema: {
+		default_value: 'any',
+	},
+	meta: {
+		width: 'half',
+		interface: 'select-dropdown',
+		options: {
+			choices: [
+				{ text: 'Any', value: 'any' },
+				{ text: 'Delivered', value: 'delivered' },
+				{ text: 'Expired', value: 'expired' },
+				{ text: 'Failed', value: 'failed' },
+				{ text: 'Rejected', value: 'rejected' },
+				{ text: 'Accepted', value: 'accepted' },
+				{ text: 'buffered', value: 'buffered' },
+				{ text: 'Unknown', value: 'unknown' },
+				{ text: 'Deleted', value: 'deleted' }
+			]
+		}
+	}
+},
+```
+
+After the `options` section, there is the ability to limit the width and height of the panel. Since this panel will hold
+a lot of data, set these to `24` for the width and `18` for the height:
+
+```js
+minWidth: 24,
+minHeight: 18,
+```
+
+The output of these options will look like this:
+
+<img src="https://marketing.directus.app/assets/053298df-a5ff-4fd5-85ec-ff0f78bb64ca.png" alt="Form shows product type dropdown, direction dropdown, date range dropdown, included message checkbox, and status dropdown." style="padding: 2px 12px;">
+
+## Prepare the View
+
+Open the `panel.vue` file and you will see the starter template and script. Skip to the script section and import the
+following packages:
+
+```js
+import { useApi } from '@directus/extensions-sdk';
+import { adjustDate } from '@directus/shared/utils';
```

Comment:
Is this still accurate? IIRC that's now in `@directus/utils`

Reply:
I don't know the answer to this. Paging @Nitwel or @paescuj for confirmation, and then I'll make the edit (or not)

Reply:
Yup, that's in `@directus/utils`:
```suggestion
import { adjustDate } from '@directus/utils/shared';
```